peps/pep-0755.rst

```diff
@@ -1,7 +1,7 @@
 PEP: 755
 Title: Status quo of the CPython 3.12 build system
 Author: Erlend Egeberg Aasland <erlend@python.org>,
-        Zachary Ware <zachary.ware@gmail.com>
+        Zachary Ware <zach@python.org>
```

Comment:
BTW, you'll need to fix your email in your previous PEPs first. I had to do the same (see python/peps#3463)